index.html

```diff
@@ -28,7 +28,7 @@ <h1>Dr.Norman Borlaug</h1>
       <figure id="img-div" class="">
         <img src="./image-asset.jpeg" alt="" id="image" class="img-fluid" />
```

Comment:
Please give alt some values. The required alt attribute specifies an alternate text for an image if the image cannot be displayed.